### PR TITLE
gui(settings): add a button to copy auth fields & mask password

### LINKED ISSUE #1160: Settings > Bitcoin Core: password should be selectable
in order regtest user to use the cli for instance

### PR DESCRIPTION
**Pull request by pythcoiner**
fixes #1160 
image

### CODE REVIEW

**Comment by pythcoiner** [comment]
done 
image

**Comment by pythcoiner** [comment]
also replaced node password by stars since now we can copy it:

image

**Comment by jp1ac4** [comment]
Perhaps for consistency there should also be a clipboard icon for the Electrum server URL?

(This needs a rebase on master due to the DB migration.)

**Comment by pythcoiner** [comment]
image
done

File: liana-gui/src/app/state/settings/bitcoind.rs [hunk 1]
```diff
@@ -361,6 +361,7 @@ impl BitcoindSettings {
                     });
                 }
             }
+            view::SettingsEditMessage::Clipboard(text) => return clipboard::write(text),
```
Comment (jp1ac4) [inline liana-gui/src/app/state/settings/bitcoind.rs:364]: This will be needed also for `ElectrumSettings`.

Comment (pythcoiner) [reply liana-gui/src/app/state/settings/bitcoind.rs:364]: done

**Review by jp1ac4 — APPROVED** [review]
Tested ACK 9679f0f2926e2bc90ba7b8b61f138dcabaf22451.

**Pull request merged**